controller/api/util/api_utils.go

```diff
@@ -268,6 +277,28 @@ func BuildEdgesRequest(p EdgesRequestParams) (*pb.EdgesRequest, error) {
 	return edgesRequest, nil
 }
 
+// BuildGatewayRequest builds a Public API GatewayRequest from a
+// GatewayRequestParams
+func BuildGatewayRequest(p GatewayRequestParams) (*pb.GatewaysRequest, error) {
+	window := defaultMetricTimeWindow
+	if p.TimeWindow != "" {
+		_, err := time.ParseDuration(p.TimeWindow)
+		if err != nil {
+			return nil, err
+		}
+		window = p.TimeWindow
+	}
+	multiClusterNameSpace := p.GatewayNamespace
+	if multiClusterNameSpace == "" {
```

Comment:
if unspecified, I think we want to leave this blank which should return gateways in all namespaces